src/victory-util/add-events.js

```diff
@@ -0,0 +1,61 @@
+import { defaults, assign, isFunction, partialRight } from "lodash";
+import Events from "./events";
+
+export default (WrappedComponent) => {
+  return class addEvents extends WrappedComponent {
+    constructor() {
+      super();
+      this.state = {};
+      const getScopedEvents = Events.getScopedEvents.bind(this);
+      this.getEvents = partialRight(Events.getEvents.bind(this), getScopedEvents);
+      this.getEventState = Events.getEventState.bind(this);
+    }
+
+    componentWillMount() {
```

Comment:
No can do. I need the static methods on the WrappedComponents to stay intact. 

I just opened a PR https://github.com/FormidableLabs/victory-core/pull/151 that should help with some of your concerns. `state` wont get nuked if it exists, and the lifecycle methods on `WrappedComponent` will still get called.